boot/assets.go

```diff
@@ -171,10 +171,10 @@ func CopyBootAssetsCacheToRoot(dstRoot string) error {
 var ErrObserverNotApplicable = errors.New("observer not applicable")
 
 // TrustedAssetsInstallObserverForModel returns a new trusted assets observer
-// for use during installation of the run mode system, provided the device model
-// supports secure boot. Otherwise, nil and ErrObserverNotApplicable is
-// returned.
-func TrustedAssetsInstallObserverForModel(model *asserts.Model, gadgetDir string) (*TrustedAssetsInstallObserver, error) {
+// for use during installation of the run mode system provided the device model
+// indicates support for trusted or managed assets. Otherwise, nil and
+// ErrObserverNotApplicable is returned.
```

Comment:
```
// TrustedAssetsInstallObserverForModel returns a new trusted assets observer
// for use during installation of the run mode system to track trusted and control managed assets, 
// provided the device model indicates this might be needed.
// Otherwise, nil and ErrObserverNotApplicable is returned.
```